packages/docs-site/.dmno/config.mts

```diff
@@ -15,5 +16,23 @@ export default defineDmnoService({
     GOOGLE_FONT_FAMILIES: {
       value: 'family=Days+One&family=Fira+Code:wght@300&family=Inter:wght@100..900'
     },
+
+    // TODO: move to netlify plugin/preset
+    CONTEXT: {
+      extends: DmnoBaseTypes.enum(['dev', 'branch-deploy', 'deploy-preview', 'production']),
+      description: 'netlify build context',
+      externalDocs: {
+        url: 'https://docs.netlify.com/configure-builds/environment-variables/#build-metadata',
+        description: 'Netlify docs',
+      },
+    },
+    DMNO_ENV: {
```

Comment:
should we move these up to the root since we're deploying multiple things on Netlify?

Reply:
yeah I figured once we formalize it a bit